webpack/assets/javascripts/react_app/components/PermissionDenied/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Button, Icon } from 'patternfly-react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../common/I18n';
+
+const PermissionDenied = ({ missingPermissions }) => {
+  const permissionsList = missingPermissions.map(permission => (
+    <li key={permission}>
+      <strong>{permission}</strong>
+    </li>
+  ));
+
+  return (
+    <div className="blank-slate-pf">
+      <div className="blank-slate-pf-icon">
+        <Icon name="lock" color="#9c9c9c" size="2x" title="lock" />
+      </div>
+      {__('You are not authorized to perform this action.')}
```

Comment:
It would be nice if we could override the text using props.
See how `DocumentationLink` using `defaultProps`:
https://github.com/theforeman/foreman/blob/a91f1683488c11f51634d02a7c0245fcc85725b5/webpack/assets/javascripts/react_app/components/common/DocumentationLink/index.js

IMO all the text values should be configured with `defaultProps`.

Reply:
I would agrue here on usefullness of such props in this component... will you ever want to customize them in here?
I like the concept in general, so I'm ok eighter way :man_shrugging: but we would at least have the strings all together, I would not use `children` for it though, at least not in case of this component, as it would be pretty missleading with multiple strings?

Reply:
IMO we should always use translations this way and I would love to see it becomes a convention.
We do use it quite a lot.

Agree it doesn't that useful in term of overriding (sometimes) but it helps with keeping all the translations together.

+1 for not using the `children`

Reply:
@sharvit @ezr-ondrej Thanks, I like this idea!